clang/docs/PointerAuthentication.rst

```diff
@@ -354,6 +354,23 @@ Given that ``signedPointer`` matches the layout for signed pointers signed with
 the given key, extract the raw pointer from it.  This operation does not trap
 and cannot fail, even if the pointer is not validly signed.
 
+``ptrauth_sign_constant``
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: c
+
+  ptrauth_sign_constant(pointer, key, discriminator)
+
+Return a signed pointer for a constant address in a manner which guarantees
+a non-attackable sequence.
```

Comment:
It would be nice to have this term described more deeply. Do you mean an absence of a signing oracle when talking about a non-attackable sequence?

Reply:
Later additions to this document describe that in depth, you can look for
> [clang][docs] Document the ptrauth security model.

on my branch

Reply:
Thanks, now I see that. Are you going to submit documentation changes all at once as a separate PR? If it's not too time-consuming, I'd prefer having all related terms defined as a part of this PR (particularly, at least the part "Attackable code sequences"), but I'm OK with submitting documentation changes separately.